code/modules/food_and_drink/sandwiches.dm

```diff
@@ -137,6 +161,14 @@
 		food_effects = list("food_hp_up_big")
 		meal_time_flags = MEAL_TIME_FORBIDDEN_TREAT
 
+	spooky_blt
+		name = "haunted BLT"
```

Comment:
```suggestion
		name = "Boo-LT"
```

Reply:
it's punny!

Reply:
10,000% yes, I couldn't think of a good name at the time anyways lmao